docs/source/features/playground.md

```diff
@@ -0,0 +1,45 @@
+---
+title: GraphQL Playground
```

Comment:
I don't think this should be a feature on its own.  Enabling playground in production looks like a small tip/trick that we can add to `what's new` section.

I'd like to know @abernix's thoughts on this though